e2e/benchmarks/benchmark_util.js

```diff
@@ -553,6 +557,40 @@ async function setEnvFlags(flagConfig) {
   }
 }
 
+/**
+ * Set flags from URL. URL should be in the format:
+ * ?tfjsflags=FLAG1:1,FLAG2:true.
+ */
+function setEnvFlagsFromUrlParameters() {
+  const TENSORFLOWJS_FLAGS_PREFIX = 'tfjsflags';
```

Comment:
Maybe it's better to call it envflags?

Reply:
In tfjs-core, this is also called tfjsflags.

Reply:
Then I'm fine.